lib/scanner/read_chars.py

```diff
@@ -0,0 +1,341 @@
+import cv2
```

Comment:
Add a shebang `#!/usr/bin/env python3` to the top of this file. Make sure the permissions are set to owner-executable when the file is committed. This will allow you to run the file as `./lib/scanner/read_chars.py` directly, without writing `python3` in the shell.